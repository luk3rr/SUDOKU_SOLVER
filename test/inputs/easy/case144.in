005080406 000070100 201005790 000900000 036800000 040000020 020004060 000700041 010003270 
