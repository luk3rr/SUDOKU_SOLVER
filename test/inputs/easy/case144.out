Solving the following grid:
0 0 5 | 0 8 0 | 4 0 6 
0 0 0 | 0 7 0 | 1 0 0 
2 0 1 | 0 0 5 | 7 9 0 
------+-------+------
0 0 0 | 9 0 0 | 0 0 0 
0 3 6 | 8 0 0 | 0 0 0 
0 4 0 | 0 0 0 | 0 2 0 
------+-------+------
0 2 0 | 0 0 4 | 0 6 0 
0 0 0 | 7 0 0 | 0 4 1 
0 1 0 | 0 0 3 | 2 7 0 


Solution found :')

9 7 5 | 2 8 1 | 4 3 6 
3 8 4 | 6 7 9 | 1 5 2 
2 6 1 | 4 3 5 | 7 9 8 
------+-------+------
1 5 2 | 9 4 6 | 3 8 7 
7 3 6 | 8 5 2 | 9 1 4 
8 4 9 | 3 1 7 | 6 2 5 
------+-------+------
5 2 7 | 1 9 4 | 8 6 3 
6 9 3 | 7 2 8 | 5 4 1 
4 1 8 | 5 6 3 | 2 7 9 

Algorithm: BFS
Total time: 47 ms
Total expanded states: 7503
